### PR TITLE
Use hugo Local to make sure events are set to the correct timezone

### PR DESCRIPTION
**Pull request by SThomasP**

### CODE REVIEW

File: data/calendar.json
Comment (pantierra) [inline data/calendar.json:1]: Please add this file back on. Probably a newly generated one with the time information in UTC.

**Review by pantierra — COMMENTED** [review]
Love this - great improvement! And it looks already really good. Just had one comment.

File: data/calendar.json
Comment (gueldenstone) [reply data/calendar.json:1]: Sorry that is on me. I thought it was an artfefact left from the previous PR.

**Comment by pantierra** [comment]
Another tiny tiny thing: please remove the import statement in the download script and the note in README.md about `pytz`.

**Comment by gueldenstone** [comment]
Looks great thank you so much for figuring this out!

**Review by pantierra — APPROVED** [review]
Indeed, this is really good stuff. Many thanks!

**Review by pantierra — CHANGES_REQUESTED** [review]
Sorry found one thing. The current event highlight doesn't seem to work anymore. Easy to spot right now.

**Review by pantierra — APPROVED** [review]
Works well. Thanks a lot!

**Pull request merged**